src/components/Footer/Footer.js

```diff
@@ -1,21 +1,24 @@
 import React from 'react';
-import { FormattedMessage } from 'react-intl';
+import { FormattedMessage, injectIntl } from 'react-intl';
 
-const Footer = () => (
-  <footer className="footer-main">
-    <div className="footer-wrapper">
-      <span>
-        <FormattedMessage id="footer.iso" />
-      </span>
-      <span>
-        &copy; {new Date().getFullYear()} Doppler LLC.{' '}
-        <FormattedMessage id="footer.allRightReserved" />.{' '}
-        <a className="link--default" href="https://fromdoppler.com/privacidad">
-          <FormattedMessage id="footer.privacy" />
-        </a>
-      </span>
-    </div>
-  </footer>
-);
+const Footer = (props) => {
+  const { formatMessage } = props.intl;
+  return (
+    <footer className="footer-main">
+      <div className="footer-wrapper">
+        <span>
+          <FormattedMessage id="footer.iso" />
+        </span>
+        <span>
+          &copy; {new Date().getFullYear()} Doppler LLC.{' '}
+          <FormattedMessage id="footer.allRightReserved" />.{' '}
+          <a className="link--default" href={formatMessage({ id: 'footer.privacy_url' })}>
```

Comment:
@fcoronelmakingsense @GusBaamonde do we really need a class (`link--default`) for default links?

Why not have classes for **non-default** ones?

Reply:
you mean define alls `a` with green color?

Reply:
in old doppler breaks a lot of things maybe

Reply:
ok

Reply:
Solo digo que si hay un estilo de link por defecto, que no requiera clases. 

Si hay estilos diferentes (al por defecto), que si requieran clases.

Me parece algo obvio, se me debe estar escapando algo, ¿que podrá ser?

Reply:
Mi miedo es que al no conocer toda la aplicacion repercuta de forma negativa. Siguiendo la linea de @andresmoschini podemos cambiar la clase a "link--footer" y a medida que avancemos podemos cambiar los links con estilo por default. Les parece?

Reply:
> al no conocer toda la aplicacion repercuta de forma negativa
Eso debe ser lo que se me está escapando, podés darme un ejemplo de uno de esos posibles problemas?

Reply:
Y... quizas algun link que esten usando dentro de una tabla que no tenga estilo y tome el "default" y le cambie el estilo a la tabla o en algun lugar suelto. No lo se.

Reply:
Te referís al Doppler _Legacy_, correcto? en esos casos por ahí podríamos hacer un parche solo para Doppler, que opinás?

Reply:
Si me parece bien!
Entonces pongo un ticket para cambiar el "link--default" y coloco este estilo como default de todos los links?

Reply:
A mi me gusta, si no hay objeciones iría por ahí.